src/app-layout/utils/use-resize.tsx

```diff
@@ -2,17 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 import React, { useState, useEffect } from 'react';
 
-import clsx from 'clsx';
-
-import ResizeHandler from '../../split-panel/icons/resize-handler';
+import PanelResizeHandle from '../../internal/components/panel-resize-handle';
 import { getLimitedValue } from '../../split-panel/utils/size-utils';
 import { usePointerEvents } from './use-pointer-events';
 import { useKeyboardEvents } from './use-keyboard-events';
 import { AppLayoutProps } from '../interfaces';
 
-import splitPanelStyles from '../../split-panel/styles.css.js';
 import testutilStyles from '../test-classes/styles.css.js';
-import styles from '../visual-refresh/styles.css.js';
```

Comment:
`src/app-layout/utils` is a folder to reuse code between refresh and classic. It is very unexpected to see a refresh-only styles import here